quick_tour/flex_recipes.rst

```diff
@@ -177,6 +177,8 @@ rich API for a ``product`` table? Create a ``Product`` entity and give it the
 
     // src/Entity/Product.php
     // ...
```

Comment:
This expression makes explicit that there are some statements which are not exposed at this point in the file.

Reply:
Although you are right, this is the "Quick Tour" docs, so it's designed for the absolute Symfony newcomer. I think it's better if we show all `use` and all `namesapces` in the examples of this section. Let's merge this and I'll review the other examples in a separate PR.